sdk/core/Azure.Core/src/DiagnosticsOptions.cs

```diff
@@ -81,10 +83,15 @@ internal DiagnosticsOptions(DiagnosticsOptions? diagnosticsOptions)
         public bool IsLoggingEnabled { get; set; } = true;
 
         /// <summary>
-        /// Gets or sets value indicating whether distributed tracing spans are going to be created for this clients methods calls and HTTP calls.
+        /// Gets or sets value indicating whether distributed tracing spans are going to be created for the clients methods calls and HTTP calls.
         /// </summary>
         public bool IsDistributedTracingEnabled { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets value indicating whether distributed tracing spans created for inner public method calls should be suppressed.
+        /// </summary>
+        public bool SuppressNestedClientSpans { get; set; }
```

Comment:
I'm curious whether we actually need this - it looks like only three libraries are being updated to set this to False - does that mean those libraries have nested client spans? If they don't it may be worth just changing the default behavior rather than introducing this property.

Reply:
Thinking about it more - I can see that we do need this. Even if it doesn't impact any of our released libraries, other consumers of Core could be broken by making this a default change to the behavior.
Still, I think we may be able to get away with passing "true" for the updated libraries.